docs/intro.md

```diff
@@ -0,0 +1,52 @@
+---
+sidebar_position: 1
+---
+
+# Introduction
+
+Payjoin is a protocol for creating collaborative transactions between a sender (Alice) and a receiver (Bob). It is simple, requires no changes to bitcoin, can operate without any manual interaction from the user, and has a variety of benefits and use cases, including:
+
+- **Privacy**
+- **Cost & Time Savings**
+- **Scaling**
+- **Flexibility & Extensibility**
+
+In a typical on-chain bitcoin transaction, all of the [UTXOs](https://unchained.com/blog/what-is-a-utxo-bitcoin/) would belong to Alice. After all, one might ask, what sense does it make for Bob to send bitcoin to himself? It is _assumed_ that the spender in a transaction is the one who owns every UTXO:
```

Comment:
A link is good, but any abbreviation / acronym should be stated in full followed by the abbreviation in parens to use further down the article. e.g. "Unspent Transaction Outputs (UTXOs)." A brief explanation of how bitcoin uses UTXOs seems necessary to proceed with technical details.

What purpose does the double "After all, one might ask" serve? Would fewer words do?

Why is assumed italicized?

Reply:
> any abbreviation / acronym should be stated in full followed by the abbreviation in parens to use further down the article.
Yes! Good catch.

> A brief explanation of how bitcoin uses UTXOs seems necessary to proceed with technical details.
Can do.

> What purpose does the double "After all, one might ask" serve? Would fewer words do?
It was intended to emphasize that it's unnatural to think of UTXOs belonging to more than one owner and contrasts that with the previous sentence as being the natural thing to do. Just saying "What sense does it make..." feels to not get enough of the point across to me that the reader would also naturally think Bob wouldn't send the bitcoin to himself just like a chain analysis company. I agree that "one might ask" is over the top though, perhaps we can reduce to:
"After all, what sense does it make..."

> Why is assumed italicized?
Because I have an overzealous love of em_PHA_sis. Will remove 😄